data.go

```diff
@@ -138,6 +140,11 @@ func (m Msg) MarshalJSON() ([]byte, error) {
 		if err != nil {
 			return nil, fmt.Errorf("Failed to marshal NodeStatusDataMsg: %v", err)
 		}
+	case string:
```

Comment:
Since all the other message types have their own payloads, should we do the same here for consistency and a better chance at forward compatibility?